### DISCUSSION TITLE
Is it possible to add a conditional statement

### DISCUSSION THREAD

**Discussion by Shasti001** [nftchef/art-engine#309] [Help + Q&A]
Ok the idea is as follows..

I would like to add a conditional if statement so do the following: 

If layer [hat] =  [hat-01] then layer [hair] is false.  

Sorry have I have limited coding skills but a basic understanding.

**Comment by nftchef** [comment]
this is supported as part of the config, either through organizing the files into sub-folders https://generator.nftchef.dev/readme/branching-if-then

 or using the `forcedCombinatioins` option
 https://generator.nftchef.dev/readme/inclusions-and-exclusions#forced-combinations
